controllers/result.go

```diff
@@ -22,10 +24,18 @@ func (c *ResultController) URLMapping() {
 
 func (c *ResultController) List() {
 	c.EnsureAuthenticatedUser()
-
 	c.TplName = "results/list.html"
-
 	web.ReadFromRequest(&c.Controller)
+
+	if c.CurrentUser != nil {
+		results, err := models.GetResultsByUserId(c.CurrentUser.Id)
+		if err != nil {
+			logs.Warn("Failed to get current user results", err.Error())
```

Comment:
How about separate the error message? Otherwise the error will be like `Failed to get current user resultssome-error`
```suggestion
			logs.Warn("Failed to get current user results: ", err.Error())
```

Reply:
✅  fixed in [97e86e2](https://github.com/carryall/go-google-scraper-challenge/pull/65/commits/97e86e2fec1c39097cfc178a00f8156c4ec9ed28)